tests/cli/test_suite.py

```diff
@@ -1,489 +1,1109 @@
+def test_suite_new_non_interactive_with_suite_name_prompted_default_opens_jupyter(
     mock_webbroser,
     mock_subprocess,
+    mock_emit,
     caplog,
     monkeypatch,
-    data_context_parameterized_expectation_suite,
-    filesystem_csv_2,
+    titanic_pandas_data_context_with_v013_datasource_with_checkpoints_v1_with_empty_store_stats_enabled,
 ):
-    """
-    Running "suite new" should:
-    - make an empty suite
-    - open jupyter
-    - NOT open data docs
-    """
-    context = data_context_parameterized_expectation_suite
-    project_root_dir = context.root_directory
-    os.mkdir(os.path.join(project_root_dir, "uncommitted"))
-    root_dir = project_root_dir
-    runner = CliRunner(mix_stderr=False)
-    csv = os.path.join(filesystem_csv_2, "f1.csv")
+    context: DataContext = titanic_pandas_data_context_with_v013_datasource_with_checkpoints_v1_with_empty_store_stats_enabled
     monkeypatch.chdir(os.path.dirname(context.root_directory))
-    result = runner.invoke(
+
+    project_dir: str = context.root_directory
+    uncommitted_dir: str = os.path.join(project_dir, "uncommitted")
+
+    expectation_suite_name: str = "warning"
+
+    runner: CliRunner = CliRunner(mix_stderr=False)
+    result: Result = runner.invoke(
         cli,
-        ["--v3-api", "suite", "new", "--suite", "foo"],
-        input=f"{csv}\n",
+        f"--v3-api suite new",
+        input="\n",
         catch_exceptions=False,
     )
-    stdout = result.stdout
-
     assert result.exit_code == 0
-    assert "Enter the path" in stdout
-    assert "Name the new expectation suite" not in stdout
-    assert (
-        "Great Expectations will choose a couple of columns and generate expectations"
-        not in stdout
+
+    stdout: str = result.stdout
+    assert "Select a datasource" not in stdout
+    assert f"Name the new Expectation Suite [warning]:" in stdout
+    assert "Opening a notebook for you now to edit your expectation suite!" in stdout
+    assert "If you wish to avoid this you can add the `--no-jupyter` flag." in stdout
+
+    expected_suite_path: str = os.path.join(
+        project_dir, "expectations", f"{expectation_suite_name}.json"
     )
-    assert "Generating example Expectation Suite..." not in stdout
-    assert "The following Data Docs sites were built" not in stdout
-    assert (
-        "Great Expectations will create a new Expectation Suite 'foo' and store it here"
-        in stdout
+    assert os.path.isfile(expected_suite_path)
+
+    expected_notebook_path: str = os.path.join(
+        uncommitted_dir, f"edit_{expectation_suite_name}.ipynb"
     )
-    assert (
-        "Because you requested an empty suite, we'll open a notebook for you now to edit it!"
-        in stdout
+    assert os.path.isfile(expected_notebook_path)
+
+    run_notebook(
+        notebook_path=expected_notebook_path,
+        notebook_dir=uncommitted_dir,
+        string_to_be_replaced="context.open_data_docs(resource_identifier=suite_identifier)",
```

Comment:
Very cool! We eventually need to port this back to the Checkpoint CLI.

Reply:
Should be very easy!